frontend/src/pages/projects/screens/detail/data-connections/useDataConnections.ts

```diff
@@ -23,8 +24,7 @@ const useDataConnections = (namespace?: string): FetchState<DataConnection[]> =>
       ).then((secrets) =>
         secrets
           .filter<AWSSecretKind>( // TODO: this will make more sense when we have more data connection types
-            (secret): secret is AWSSecretKind =>
-              !!secret.metadata.labels?.[KnownLabels.DATA_CONNECTION_AWS], // note, always true is fine for now
+            (secret): secret is AWSSecretKind => isSecretAWSSecretKind(secret), // note, always true is fine for now
```

Comment:
Can you not simply use the function as is?
```suggestion
            isSecretAWSSecretKind
```